packages/panels/docs/02-getting-started.md

```diff
@@ -350,11 +351,11 @@ public static function table(Table $table): Table
 }
 ```
 
-Note that for the owner column, we don't use `owner_id` as the text column name. Instead, we use `owner.name`. This dot notation is used by Filament to eager-load the results of that relationship, and render a list of owner names, instead of their IDs. You could add additional columns for the owner's email address and phone number if you wish.
+**NOTE:** Filament uses dot notation to eager-load related data. We used `owner.name` in our table to display a list of owner names instead of less informational ID numbers. You could also add columns for the owner's email address and phone number.
```

Comment:
```suggestion
> Filament uses dot notation to eager-load related data. We used `owner.name` in our table to display a list of owner names instead of less informational ID numbers. You could also add columns for the owner's email address and phone number.
```